corehq/apps/hqcase/tests/test_utils.py

```diff
@@ -112,3 +116,20 @@ def get_case(*args, **kwargs):
         yield case
     finally:
         factory.close_case(case.case_id)
+
+
+class TestIsCopiedCase(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.case_1 = create_case(domain=DOMAIN, name='case_1', case_json={'type': 'original'})
+        cls.case_2 = create_case(
+            domain=DOMAIN,
+            name='case_2',
+            case_json={'type': 'copied', CaseCopier.COMMCARE_CASE_COPY_PROPERTY_NAME: cls.case_1.case_id}
+        )
```

Comment:
Do we need this `setUpClass` method? It doesn't look like the current tests require this additional setup. In the future, if we have common setup code, I think we could look at using `setUp` first. `setUpClass` introduces the possibility that we mutate common data in between tests, which leads to situations where the order tests run matters -- and can be very frustrating to debug. I think we mostly want to use `setUpClass` when we've witnessed performance issues that can't be resolved in another way. Here, these tests are running super fast, so it probably isn't necessary

Reply:
Noted on this. The only reason for having in `setupClass` was `case_2` being a `copy_1` which could be handled in a different way.